### ISSUE TITLE
Not sure if xivlauncher is updating itself

### ISSUE THREAD

**Issue by urbanracer34** [goatcorp/FFXIVQuickLauncher#1531]
### Update disclaimer

- [X] Yes, I have checked and my issue is not related to the game updating and plugins not working correctly.

### What did you do?

First off I am on an Steam Deck. 

I am having trouble determining if my XIVLauncher has updated.  I've updated FFXIV to the latest version today.

https://github.com/goatcorp/FFXIVQuickLauncher shows version 6.4.0 HOWEVER https://github.com/goatcorp/XIVLauncher.Core is still on version 1.0.8. Same for the Discover store.

Which one do I have to update? Or is it done already because I could download the patches? 

### Platform

Linux (native)

### Wine/Proton runner version

UNSURE

**Comment by tommadness** [comment]
XIVLauncher (Windows) is a separate project from XLCore (Linux/Multiplatform). You are currently on the most up to date XLCore. We will be updating it for FFXIV 7.0, please stand by.

**Comment by urbanracer34** [comment]
So https://github.com/goatcorp/XIVLauncher.Core is the one getting an update?

**Comment by tommadness** [comment]
Yes. That project will be updated to 1.0.0.9 to accommodate the Dawntrail update.
